esmvalcore/_main.py

```diff
@@ -47,10 +49,28 @@
 """ + __doc__
 
 
+def parse_resume(resume, recipe):
+    """Set `resume` to a correct value and sanity check."""
+    if not resume:
+        return []
+    if isinstance(resume, str):
+        resume = resume.split(' ')
```

Comment:
here: we should say don't separate by comma (as per me previous comment) or do a `try` see if they are indeed separated by comma (you got no idea how many times users separate stuff by comma coz that's so normal :grin: )

Reply:
But what if they have a comma in the path? At least you're not supposed to add spaces in linux paths.

Reply:
if they have a comma in their path I will personally commission a missile strike on them :rocket: :laughing: